src/config/Config.zig

```diff
@@ -1086,7 +1086,7 @@ keybind: Keybinds = .{},
 /// will appear in the logs.
 ///
 /// Changing this configuration at runtime will only affect new terminals, i.e.
-/// new windows, tabs, etc.
+/// new windows, tabs, etc. A restart is necessary for existing windows.
```

Comment:
Doesn't the previous sentence imply this? You don't need to restart Ghostty, you only need to recreate the windows.